sw/host/opentitantool/src/backend/mod.rs

```diff
@@ -50,6 +51,7 @@ pub fn create(args: &BackendOpts) -> Result<TransportWrapper> {
         "verilator" => verilator::create(&args.verilator_opts),
         "ultradebug" => ultradebug::create(args),
         "cw310" => cw310::create(args),
+        "ti50_host_emulation" => ti50::create(args),
```

Comment:
Per discussion, I think we were going to rename this `host_emulation`.

Reply:
Done